src/RootCloseWrapper.js

```diff
@@ -10,6 +10,14 @@ const CLICK_WAS_INSIDE = '__click_was_inside';
 
 let counter = 0;
 
+function isLeftClickEvent(event) {
+  return event.button === 0
```

Comment:
Why so strict when semicolon insertion keeps you safe? :-)
http://www.ecma-international.org/ecma-262/5.1/#sec-7.9